### DISCUSSION TITLE
Resume Functionality for Various Players (Video, PDF, ePub, YouTube)

### DISCUSSION THREAD

**Discussion by rushi-tekdi** [Sunbird-Knowlg/Community#227] [Q&A]
We require a resume feature for the video, PDF, ePub, and YouTube players so that users can continue from where they last left off. For example:

- If a user closes a video at the 2-minute mark in a 5-minute video, they should resume from the 2-minute mark upon reopening.
- If a user closes a PDF on page 4, they should reopen from page 4.
- The same behavior should apply to the ePub and YouTube players.

Could you confirm if this functionality is available in these players? If so, please specify which key should be configured to enable this feature.

GitHub Repositories:
[Sunbird Video Player](https://github.com/Sunbird-Knowlg/sunbird-video-player/)
[Sunbird PDF Player](https://github.com/Sunbird-Knowlg/sunbird-pdf-player/)
[Sunbird ePub Player](https://github.com/Sunbird-Knowlg/sunbird-epub-player/)
[Sunbird Content Player](https://github.com/Sunbird-Knowlg/sunbird-content-player/)

@coolbung @pallakartheekreddy @snehal0904

**Comment by pallakartheekreddy** [comment]
@rushi-tekdi Currently, we don't have support for the resume feature in Sunbird players.
